content_schemas/formats/worldwide_organisation.jsonnet

```diff
@@ -4,6 +4,26 @@
       type: "object",
       additionalProperties: false,
       properties: {
+        ordered_corporate_information_pages: {
+          type: "array",
+          items: {
+            type: "object",
+            additionalProperties: false,
+            required: [
+              "title",
+              "href",
+            ],
+            properties: {
+              title: {
+                type: "string",
+              },
+              href: {
```

Comment:
I'm probably drifting into the optimistic here, but are these corporate information pages available via the links hash? I'm a bit concerned that title and href are rather delicate attributes that have a tight coupling.

I'm wondering if this could be done entirely through expanded links, assuming title is not customised and if title is customised here whether the href could be a content_id instead so the href can be pulled out of expanded links

Reply:
Unfortunately the title comes from Whitehall arbitrarily defining what to include here (sometimes it doesn't match the name of the document, being pulled in from the locale files).  It's the same for organisation pages too.

Reply:
Sure, that seems fine, would the href be able to be a content_id then so the document can be matched from the links rather than a hardcoded URL?

Reply:
That should be possible.  I'll have to add the pages in as links first.

Reply:
I've added these in as links and replaced the `href` in the details hash with a content ID.